web/src/Root.jsx

```diff
@@ -128,6 +128,38 @@ class Root extends Component {
     this.setState({ initSessionId: "" });
   }
 
+  getPendingApps = async () => {
+    try {
+      const res = await fetch(`${window.env.API_ENDPOINT}/pendingapps`, {
+        headers: {
+          "Authorization": Utilities.getToken(),
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      });
+      if (!res.ok) {
+        if (res.status === 401) {
+          Utilities.logoutUser();
+          return;
+        }
+        if (res.status === 404) {
```

Comment:
can we have this logged as an "unexpected status" as well? no need for this check

Reply:
this is to address a corner case where the license got deleted in the DB.